cmd/gitserver/server/server.go

```diff
@@ -2984,3 +3266,16 @@ func isAbsoluteRevision(s string) bool {
 	}
 	return true
 }
+
+type poolRepoCloneInProgress struct {
+	triedFor       int64
```

Comment:
To be more explicit:
```suggestion
	triedForSeconds       int64
```